scripts/generate-deployment-dev

```diff
@@ -23,8 +23,8 @@
 set -e
 
 # Setup
-PROD_IMAGE="$1"	# e.g. 'karydia/karydia'
-DEV_IMAGE="$2"	# e.g. 'karydia/karydia-dev'
+PROD_IMAGE="$1"	# e.g. 'eu.gcr.io/gardener-project/karydia/karydia'
```

Comment:
IMHO there is no need to change, and thus extend, this comment which just mentions an example value

Reply:
Cannot really get your point here. Why shouldn't it be changed? Why should the example value still reference an older version that is not used?

Reply:
IMHO I see no additional benefit from a change there and the example can still be the same because if we want to change to another registry anytime we have to change the comments again, and again, and again but it is just an example no binding and no production usage. This is why I suggest to let the comments as they are. And they way how to use it correctly is described in markdown files. Again no need to change an example comment.

Reply:
Done.